src/App.tsx

```diff
@@ -1,14 +1,30 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [loader, setLoader] = useState(false);
+  const [selectAll, setSelectAll] = useState('active');
```

Comment:
```suggestion
  const [filter, setFilter] = useState('active');
```
it should be dynamic value